core/src/main/java/com/novoda/noplayer/exoplayer/EventListener.java

```diff
@@ -0,0 +1,58 @@
+package com.novoda.noplayer.exoplayer;
+
+import com.google.android.exoplayer2.ExoPlaybackException;
+import com.google.android.exoplayer2.ExoPlayer;
+import com.google.android.exoplayer2.PlaybackParameters;
+import com.google.android.exoplayer2.Timeline;
+import com.google.android.exoplayer2.source.TrackGroupArray;
+import com.google.android.exoplayer2.trackselection.TrackSelectionArray;
+
+import java.util.List;
+
+class EventListener implements ExoPlayer.EventListener {
+
+    private final List<Forwarder> forwarders;
+
+    EventListener(List<Forwarder> forwarders) {
+        this.forwarders = forwarders;
+    }
+
+    @Override
+    public void onTimelineChanged(Timeline timeline, Object manifest) {
+        // no-op
```

Comment:
Because we do not know what they map to. This needs to be defined in `no-player` before it can be given to a forwarder.